pkg/ddevapp/webserver_config_assets/nginx-site-craftcms.conf

```diff
@@ -0,0 +1,81 @@
+# ddev craftcms config
```

Comment:
Was this copied from the PHP type? Do we even need to do it? (php type is used by default if this doesn't exist)

Reply:
Yes, it was copied, and then some additional changes were added (relatively minor, but nice to have). Also this way we have a default file that if we do need additional customizations, it's already there.